airbyte-webapp/src/views/layout/SideBar/SideBar.tsx

```diff
@@ -36,36 +36,99 @@ export const useCalculateSidebarStyles = () => {
   return ({ isActive }: { isActive: boolean }) => menuItemStyle(isActive);
 };
 
-const SideBar: React.FC = () => {
+interface SideBarProps {
+  additionalTopItems?: JSX.Element;
+  bottomMenuItems?: JSX.Element[];
```

Comment:
What about using `JSX.Element` for both of these? You could wrap `bottomMenuItems` in a react fragment to avoid needing to loop over an array of elements manually.